setup.py

```diff
@@ -47,6 +47,18 @@
     author_email='snakecharmers@ethereum.org',
     url='https://github.com/ethereum/eth-account',
     include_package_data=True,
+    package_data={"eth_account": ["hdaccount/wordlist/*.txt"]},
+    data_files=[('eth_account', [
+        'eth_account/hdaccount/wordlist/chinese_simplified.txt',
```

Comment:
I think you can do this via the `MANIFEST.in` file which allows for a `*` style inclusion that will not suffer from us forgetting to update this list if we add a new word-list.

https://github.com/ethpm/ethpm-cli/blob/432e3aa40faeda6eea54530bdf3d2c81fd80d5a1/MANIFEST.in#L41

Reply:
We have a `Manifest.in`, but the doctests were still failing. I _think_ we only need this for the doctests, in which case we only need the english one, so I could just include that one?

Reply:
@carver @pipermerriam So turns out if I use the generated tarball file to install `eth-account`, I need the `data_files` here, but if I use the wheel to install, I don't need data_files. How concerned are we about the tarball distribution? I assume both should work?

Reply:
Hm, this suggests that the wheel is better, all else equal: https://stackoverflow.com/a/42226284/8412986

Based on the docs though, I can't see why we would need `data_files` *and* `package_data`. So you're saying a wheel install will fail when this `data_files` entry is removed?

Reply:
Yeah, it's entirely possible I'm doing something wrong, but when I do 
```
make dist # in my eth-account dir, 
cd path/to/web3 
pip install /path/to/eth-account/eth_account/dist/eth-account-0.5.0.tar.gz
```
and open up ipython to test, I get the error where it doesn't load any of the languages from eth-account's `Mnemonic.list_languages` (code is found [here](https://github.com/ethereum/eth-account/blob/master/eth_account/hdaccount/mnemonic.py#L85-L86)). The `make dist` output shows that the files get added. My one thought was that maybe there is something going on with relative/absolute paths in the `list_languages` method?

Reply:
Got it! The syntax was wrong in the `MANIFEST.in` file